game/views.py

```
@@ -805,6 +824,30 @@ def post(self, request, *args, **kwargs):
+                email=email
+            ).first()
+
+            from django.contrib.auth.forms import (
+                PasswordResetForm
+            )
+
+            class SingleUserPasswordResetForm(
+                PasswordResetForm
+            ):
+
+                def get_users(self, email):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `get_users`

(ANN202)